help.md

```diff
@@ -73,6 +73,8 @@ If you're interested in discussing the development of the following plugins, som
 
 * [**pulp-deb** on Matrix](https://matrix.to/#/#pulp-debian:matrix.org?via=libera.chat&via=matrix.org)
 
+* [**pulp-installer** on Matrix](https://matrix.to/#/#pulp-installer:matrix.org?via=libera.chat&via=matrix.org)
```

Comment:
```suggestion
* [**pulp-installer** on Matrix](https://matrix.to/#/#pulp-installer:matrix.org)
```
I think this will do! I see it's wrong on line 74 also. I will fix that later :)